lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -768,6 +768,51 @@ public function get_stylesheet( $types = array( 'variables', 'styles', 'presets'
 		return $stylesheet;
 	}
 
+	/**
+	 * Returns a filtered declarations array if there is a separator block with only a background
+	 * style defined on theme.json by adding a color attribute to reflect the changes in the front.
+	 *
+	 * @param array $declarations List of declarations.
+	 *
+	 * @return array $declarations List of declarations filtered.
+	 */
+	private function update_separator_declarations( $declarations ) {
```

Comment:
Should be marked `static` I think:

```suggestion
	private static function update_separator_declarations( $declarations ) {
```